pkgs/applications/video/filebot/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, openjdk8, makeWrapper, autoPatchelfHook
+
+, zlib, libzen, libmediainfo, curl, libmms, glib
+}:
+
+let
+  curlWithGnuTls = curl.override { gnutlsSupport = true; sslSupport = false; };
+
+in
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "filebot";
+  version = "4.8.5";
+
+  src = fetchurl {
+    # This is a closed-source application, hence downloading a binary package.
+    url = "https://get.filebot.net/filebot/FileBot_4.8.5/FileBot_4.8.5-portable.tar.xz";
+    sha256 = "1c2a97rkv57cvy4arnmw9f3zdvxwb5zfvr9iqgpn9rkda00f2hy8";
+  };
+
+  unpackPhase = ''
+    pwd
+    tar xvf $src
+    ls -lta
+  '';
+
+  nativeBuildInputs = [ makeWrapper autoPatchelfHook
+                        # The following are lib dependencies for autoPatchelfHook to work.
+                        stdenv.cc.cc.lib zlib libzen libmediainfo curlWithGnuTls libmms glib ];
+
+  buildPhase = '''';
+  installPhase = ''
+    mkdir -p $out
+    cp -r lib/ jar/ $out/
+    mkdir -p $out/bin
+    # Keeping original location of 'filebot.sh' file relative to other files in the package due to assumptions in the script.
+    cp filebot.sh $out/filebot.sh
```

Comment:
To somewhat respect the FHS in $out, you could put everything in `$out/opt` and then link `$out/bin/filebot.sh` to `$out/opt/filebot.sh`.

Reply:
Great idea! I'm onto it